mypy/test/testcmdline.py

```diff
@@ -58,6 +58,13 @@ def test_python_cmdline(testcase: DataDrivenTestCase) -> None:
     outb = process.stdout.read()
     # Split output into lines.
     out = [s.rstrip('\n\r') for s in str(outb, 'utf8').splitlines()]
+
+    if "PYCHARM_HOSTED" in os.environ:
+        pos = next((p for p, i in enumerate(out) if i.startswith('pydev debugger: ')), None)
+        if pos is not None:
+            del out[pos]  # the attaching debugger message itself
+            del out[pos]  # plus the extra new line added
```

Comment:
This code looks quite fragile (what if the debugger output format changes?). Is it possible to configure the debugger to just not print these lines?

Reply:
Also please don't rebase your PR, merge when needed, otherwise it is hard to follow.

Reply:
we'll need to follow up then and amend it; I haven't found a way to disable it sadly :cry:

Reply:
OK.